### PR TITLE
feature/jquery-router

### PR DESCRIPTION
**Pull request by stripedpajamas**
- Separated things into folders for better visibility
- Changed template structure to export head separately
- Changed CSS to export directly
- Put weather route into a controller
- Implemented a nearly full-featured jQuery router
  -- Index page capable
  -- 404 page capable
  -- File based routes capable (e.g. stylesheets/style.css)
  -- Nested routes capable
  -- Routes live in the DOM but don't muddy the body
  -- All wrapped in jQuery events and triggers, merry Christmas

### CODE REVIEW

**Comment by nathanjohnson320** [comment]
🥇

File: router.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const $ = require('jquery')(dom.window);
+const weatherController = require('./controllers/weatherController');
+const css = require('./assets/css');
+
+// Create parent routes
+$('html').append('<index>');
+$('index')
+  .append('<not-found>')
+  .append('<stylesheets>');
+$('stylesheets')
```
Comment (nathanjohnson320) [inline router.js]: does .append('<stylesheets>').append('<style.css') work?

Comment (stripedpajamas) [inline router.js]: no-- I found that all the .appends off of index were direct children of index.

**Comment by stripedpajamas** [comment]
Moving core jQuery functionality to a different package.

**Pull request closed without merging**